src/Input/TextField/TextFieldStyle.ts

```diff
@@ -142,7 +171,19 @@ export const TextFieldInput = styled.input<TextFieldInputProps>`
       }
     }}
 
-    + ${TextFieldLabel} {
+    ${({ hasStartIcon }) => {
+      if (hasStartIcon) {
+        return `
+          + .start-icon {
```

Comment:
could we use `StartIconContainer` instead?

Reply:
Ahh right :P